k8s/helmfile/env/local/platform-nginx.nginx.conf

```diff
@@ -153,9 +164,11 @@ server {
         }
 
         location ~* "^/()(.*)" {
+                auth_request /version;
```

Comment:
statement: I think we talked about this way back when but I couldn't find a link to it. An alternative to making an auth request each time might be to use the nginx lua setup. This usecase is one of the ones they acknowledge in it: https://github.com/openresty/lua-nginx-module?tab=readme-ov-file#typical-uses:~:text=fetching%20backend%20information%20from%20external%20storage%20backends%20(like%20redis%2C%20memcached%2C%20mysql%2C%20postgresql)%20and%20use%20that%20information%20to%20choose%20which%20upstream%20backend%20to%20access%20on%2Dthe%2Dfly%2C

no drama if you don't want to change this though; it might be something we need to do if we find that the version endpoint is being hit too regularly

Reply:
smart! That's a good idea